client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1718,6 +1718,25 @@ Undocumented.prototype.changeTheme = function ( siteSlug, data, fn ) {
 	);
 };
 
+/*
+ * Change the theme and design of a given site.
+ *
+ * @param {string} [siteSlug]
+ * @param {string} [data]
+ * @param {Function} fn
+ */
+Undocumented.prototype.changeThemeAndDesign = function ( siteSlug, data, fn ) {
```

Comment:
The "undocumented" API is deprecated: p4TIVU-9KX-p2
We can just call `wpcom.req.post()` directly from `setThemeAndDesignOnSite()`